### PR TITLE
ACM-14356: Edge Management dev-preview info

### PR DESCRIPTION
**Pull request by oglok**
Description of Edge Management (project Flight Control) for ACM 2.12 dev-preview.

### CODE REVIEW

**Comment by oglok** [comment]
https://issues.redhat.com/browse/ACM-12519

File: dev-preview/README.md [hunk 1]
```diff
@@ -81,6 +84,19 @@ Follow the installation instructions in the above to install MCO operator, and e
 
 Simply set `InstanceSize` field on MCO CR to a value that would suit your monitoring needs.
 
+## Edge Management
+
+Edge Management aims to provide simple, scalable, and secure management of edge devices and applications. Users declare the operating system version, host configuration, and set of applications they want to run on an individual device or a whole fleet of devices, and Edge Management will roll out the target configuration to devices where a device agent will automatically apply them and report progress and health status back up.
```
Comment (avishayt) [inline dev-preview/README.md:89]: I would mention MicroShift/RHDE up front.

**Review by mangelajo — APPROVED** [review]
/lgtm

File: dev-preview/README.md (hunk 1 above)
Comment (fzdarsky) [inline dev-preview/README.md:89]:
```suggestion
Edge Management aims to provide simple, scalable, and secure management of edge servers or devices running image-mode RHEL / RHDE and applications on Podman or MicroShift. Users declare the target operating system version, host configuration, and set of applications they want to run similarly to how they would on edge clusters with OpenShift. Edge Management further debuts fleet management, enabling users to specify device templates that Edge Management automatically rolls out to groups of devices and reports progress and health status on.
```

**Review by mangelajo — CHANGES_REQUESTED** [review]
see @fzdarsky comments

**Comment by oglok** [comment]
Comments addressed! Thanks @avishayt @fzdarsky @mangelajo

**Review by jnpacker — APPROVED** [review]
Looks good to me.

**Pull request merged**